src/Microsoft.Data.Analysis/GroupBy.cs

```diff
@@ -28,16 +28,16 @@ public abstract class GroupBy
         /// <summary>
         /// Returns the first numberOfRows rows of each group
         /// </summary>
-        /// <param name="numberOfRows"></param>
+        /// <param name="numberOfRowsInEachGroup"></param>
         /// <returns></returns>
-        public abstract DataFrame Head(int numberOfRows);
+        public abstract DataFrame Head(int numberOfRowsInEachGroup);
```

Comment:
I kind of think `numberOfRows` is better. If you are keeping it as `numberOfRowsInEachGroup`, you need to update the summary comments. But I would keep it `numberOfRows`.

Reply:
I'll stick to `numberOfRows` for now. For all I know, this might come up in API review